### PR TITLE
Add `targetPercent` as alternative to `target`

### PR DESCRIPTION
**Pull request by mmazur**
Implements #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,7 +252,8 @@ objectives:
   - displayName: string # optional
     op: lte | gte | lt | gt # conditional operator used to compare the SLI against the value. Only needed when using a thresholdMetric
     value: numeric # optional, value used to compare threshold metrics. Only needed when using a thresholdMetric
-    target: numeric [0.0, 1.0) # budget target for given objective of the SLO
+    target: numeric [0.0, 1.0) # budget target for given objective of the SLO, can't be used with targetPercent
+    targetPercent: numeric [0.0, 100) with optional "%" suffix # budget target for given objective of the SLO, can't be used with target
```
Comment (proffalken) [inline README.md:256]: Do we need the "optional "%" suffix if the word "percent" is in the name?

Comment (mmazur) [reply README.md]: Yeah, doesn't look to pretty. Dropped it.

File: README.md [hunk 2]
```diff
@@ -273,8 +276,13 @@ objectives:
 - **value** *numeric*, required field, used to compare values gathered from
   metric source. Only needed when using a `thresholdMetric`.
 
+Either `target` or `targetPercent` must be used.
+
 - **target** *numeric [0.0, 1.0)*, required, budget target for given objective
-  of the SLO
+  of the SLO. A `target: 0.9995` is equivalent to `targetPercent: 99.95`.
```
Comment (kenfinnigan) [inline README.md:282]: Should we add a comment to `target` and `targetPercent` in the description about choosing one or the other?

At the moment it appears they're both required, unless someone sees the note on line 256?

Comment (mmazur) [reply README.md]: Updated descriptions of both to make it explicit how they must be used.

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**